src/AutoRegisterInject/Generator.cs

```diff
@@ -97,7 +124,8 @@ private static AutoRegisteredClass GetAutoRegisteredClassDeclarations(GeneratorS
 
                     return new AutoRegisteredClass(typeName,
                         registrationType,
-                        registerAs);
+                        registerAs,
+                        string.Empty);
```

Comment:
In progress. Will add / support the constructor's new argument to define a key for Keyed methods and TryAddKeyed methods